src/screens/DiscoveryScreenV2/fragment/SearchFragment.js

```diff
@@ -1,20 +1,22 @@
 import * as React from 'react';
-import { ScrollView, StyleSheet } from 'react-native';
+import {ScrollView, StyleSheet} from 'react-native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import RecentSearch from '../elements/RecentSearch';
-import { colors } from '../../../utils/colors';
+import {COLORS} from '../../../utils/theme';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to use `COLORS` from the `theme` module instead of `colors` from the `colors` module. This change is consistent with the PR's goal of standardizing color usage across the application.
```diff
- import { colors } from '../../../utils/colors';
+ import {COLORS} from '../../../utils/theme';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->